nms/app/packages/magmalte/app/views/equipment/FEGGatewaySummary.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {DataRows} from '../../components/DataGrid';
+import type {federation_gateway} from '@fbcnms/magma-api';
+
+import DataGrid from '../../components/DataGrid';
+import React from 'react';
+
+/**
+ * Returns the federation gateway description, id, hardware uuid, and it's
+ * version.
+ * @param {federation_gateway} gwInfo The federation gateway being looked at.
+ */
+export default function GatewaySummary({gwInfo}: {gwInfo: federation_gateway}) {
+  const version = gwInfo.status?.platform_info?.packages?.[0]?.version;
+
+  const data: DataRows[] = [
+    [
+      {
+        category: 'Name',
+        value: gwInfo.name,
+      },
+    ],
+    [
+      {
+        category: 'Gateway ID',
+        value: gwInfo.id,
+      },
+    ],
+    [
+      {
+        category: 'Hardware UUID',
+        value: gwInfo.device.hardware_id,
+      },
+    ],
+    [
+      {
+        category: 'Version',
+        value: version ?? 'null',
```

Comment:
A default value of `Unknown` or something like that would be more helpful to a user.

Reply:
Ok will do that, but I just wanted it to be the same as the lte network because it displays null there.